examples/FNCS/fncs_example/agent.py

```diff
@@ -44,9 +44,10 @@ def fncs_example(config_path, **kwargs):
     time_delta = config.get("time_delta", "1s")
     sim_length = config.get("sim_length", "60s")
     stop_agent_when_sim_complete = config.get("stop_agent_when_sim_complete", False)
+    subscription_topic = config.get("subscription_topic", None)
     return FncsExample(topic_mapping=topic_mapping, federate_name=federate, broker_location=broker_location,
                        time_delta=time_delta, sim_length=sim_length,
-                       stop_agent_when_sim_complete=stop_agent_when_sim_complete, **kwargs)
+                       stop_agent_when_sim_complete=stop_agent_when_sim_complete,subscription_topic, **kwargs)
```

Comment:
This line is an error because there isn't a default set for subscription_topic and there is on the previous arguments.

Reply:
SyntaxError: non-keyword arg after keyword arg